<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemInfo</class>
 <widget class="QDialog" name="DialogSystemInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System information</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelNme">
           <property name="text">
            <string>Program name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNameProgramArguments">
           <property name="text">
            <string>Program arguments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNameProgramLocation">
           <property name="text">
            <string>Program Location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNameLanguageDir">
           <property name="text">
            <string>Language dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVersion">
           <property name="text">
            <string>Program version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelGitVersion">
           <property name="text">
            <string>Git version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDateCompilation">
           <property name="text">
            <string>Compilation date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPid">
           <property name="text">
            <string>Program Pid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelConfiguration">
           <property name="text">
            <string>Configuration file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDatabase">
           <property name="text">
            <string>Database name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditProgramArguments">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLocation">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLanguageDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVersion">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGitVersion">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDateCompilation">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPid">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditConfiguration">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDatabase">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogSystemInfo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
